tests/parametric/test_headers_tracestate_dd.py

```diff
@@ -330,6 +330,10 @@ def test_headers_tracestate_dd_propagate_origin(self, test_agent, test_library):
         reason="False Bug: header[3,6]: can't guarantee the order of strings in the tracestate since they came from the map. BUG: header[4,5]: w3cTraceID shouldn't be present",
     )
     @bug(context.library in ["python@2.7.2", "python@2.7.3"], reason="AIT-9945")
+    @missing_feature(
+        context.library == "ruby",
+        reason="_dd.p.dm is reset to DEFAULT because we made the sampling decision APMAPI-812",
+    )
```

Comment:
Same, if it's a bug then : 

```suggestion
    @bug(context.library == "ruby", reason="APMAPI-812")
```

Otherwise, just ignore 😄